### PR TITLE
Make [!VIDEO] output correctly responsive

### PR DESCRIPTION
**Pull request by DuncanmaMSFT**
Updating the rendered code for the video element to correctly handle being displayed at different widths. Previous code was a fixed 640x320 that wouldn't scale on smaller or larger width pages.

### CODE REVIEW

**Comment by superyyrrzz** [comment]
@DuncanmaMSFT Please update unit tests. @fenxu Please help review.

**Comment by fenxu** [comment]
The change is OK and can work in my test. Just need to update the test cases.

---
In reply to: [261432070](https://github.com/dotnet/docfx/pull/907#issuecomment-261432070) [](ancestors = 261432070)

File: src/docfx.website.themes/default/styles/docfx.css [hunk 1]
```diff
@@ -807,6 +807,23 @@ pre > code .line-highlight {
...
+    position: relative;
+    width: 100%;
+}
+
+div.embeddedvideo iframe {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```
Comment (hellosnow) [inline src/docfx.website.themes/default/styles/docfx.css]: is it enough for width: 100%; and height: 100%; without top/left/right/bottom?

Comment (DuncanmaMSFT) [inline src/docfx.website.themes/default/styles/docfx.css]: No, the absolute position (and top/left/right/bottom) keep it in the right spot as the container resizes.

**Comment by DuncanmaMSFT** [comment]
I updated the tests, sorry about missing that on the first commit, thanks!

**Comment by DuncanmaMSFT** [comment]
So, the tests pass in VS locally... can someone explain the failure from above?

**Comment by superyyrrzz** [comment]
Sorry this is caused by a problem in DocFX's own test case... Your change is good and I've merged it.

**Pull request merged**